tools/cp-caps/device_manager.py

```diff
@@ -0,0 +1,224 @@
+        self.__dut = self.__connect_dut()
+        self.__ref = self.__connect_reference_device()
+
+        if self.__get_debug_enabled():
+            logger = logging.getLogger()
+            logger.setLevel(logging.DEBUG)
+
+    @property
+    def dut(self) -> OTCI:
+        return self.__dut
+
+    @property
+    def ref(self) -> OTCI:
+        return self.__ref
+
+    def output_test_result_string(self, name: str, value: str, align_length: int = DEFAULT_FORMAT_ALIGN_LENGTH):
+        prefix = (name + ' ').ljust(align_length, '-')
+        print(f'{prefix} {value}')
+
+    def output_test_result_bool(self, name: str, value: bool, align_length: int = DEFAULT_FORMAT_ALIGN_LENGTH):
+        self.output_test_result_string(name, 'OK' if value else 'NotSupported', align_length)
+
+    def get_default_dataset(self):
+        return self.__dut.create_dataset(channel=20, network_key='00112233445566778899aabbccddcafe')
+
+    def send_formatted_frames_retries(self, sender: OTCI, receiver: OTCI, frame: Frame, max_send_retries: int = 5):
+        for i in range(0, max_send_retries):
+            if self.__send_formatted_frame(sender, receiver, frame):
+                return True
+
+        return False
+
+    #
+    # Private methods
+    #
+    def __get_debug_enabled(self) -> bool:
+        enabled = os.getenv('DEBUG', None)
+        return enabled == 'on'
+
+    def __get_adb_key(self) -> Optional[str]:
+        return os.getenv('ADB_KEY', None)
+
+    def __connect_dut(self) -> OTCI:
+        if os.getenv('DUT_ADB_TCP'):
+            node = otci.connect_otbr_adb_tcp(os.getenv('DUT_ADB_TCP'), adb_key=self.__get_adb_key())
+        elif os.getenv('DUT_ADB_USB'):
+            node = otci.connect_otbr_adb_usb(os.getenv('DUT_ADB_USB'), adb_key=self.__get_adb_key())
+        elif os.getenv('DUT_CLI_SERIAL'):
+            node = otci.connect_cli_serial(os.getenv('DUT_CLI_SERIAL'))
+        elif os.getenv('DUT_SSH'):
+            node = otci.connect_otbr_ssh(os.getenv('DUT_SSH'))
+        else:
+            raise InvalidArgumentsError(
+                "Please set DUT_ADB_TCP, DUT_ADB_USB, DUT_CLI_SERIAL or DUT_SSH to connect to the DUT device.")
+
+        return node
+
+    def __connect_reference_device(self) -> OTCI:
+        if os.getenv('REF_CLI_SERIAL'):
+            node = otci.connect_cli_serial(os.getenv('REF_CLI_SERIAL'))
+        elif os.getenv('REF_SSH'):
+            node = otci.connect_otbr_ssh(os.getenv('REF_SSH'))
+        elif os.getenv('REF_ADB_USB'):
+            node = otci.connect_otbr_adb_usb(os.getenv('REF_ADB_USB'), adb_key=self.__get_adb_key())
+        else:
+            raise InvalidArgumentsError(
+                "Please set REF_CLI_SERIAL, REF_SSH or REF_ADB_USB to connect to the reference device.")
+
+        return node
+
+    def __send_formatted_frame(self, sender: OTCI, receiver: OTCI, frame: Frame):
+        # When the 'is_security_processed' is False, it means the frame may need the radio driver
```

Comment:
Also, it seems we use ' vs ` interchangeably. Can we standardize on backtick "`" throughout in documentation?